src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,27 @@
+// TEXT
+$medium-size-text: 12px;
+$small-size-text: 10px;
+
+//WEIGHT
+$light-weight: 400;
+$normal-weight: 500;
+$bold-weight: 700;
+
+// COLOR
+$emphasized-color: #060b35;
+$lightened-color: #616070;
+$white-color: #fff;
+$button-color: #00acdc;
+
+//TRANSITION
+$smooth-hover: color 0.3s;
```

Comment:
You can create a mixin that will take property as parameter and apply it to transition